form/src/App.js

```diff
@@ -1,19 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { render } from "react-dom";
 import Form from "@rjsf/bootstrap-4";
```

Comment:
Please start using a code linter which automatically makes sure the code style is the same everywhere. In this example there are partly spaces and partly not. Please create another PR which applies e.g. https://prettier.io/ everywhere.